src/x/time/unix_nano.go

```diff
@@ -99,7 +129,7 @@ func (u UnixNano) String() string {
 
 // Format returns the string representation for the time with the given format.
 func (u UnixNano) Format(blockTimeFormat string) string {
-	return u.ToTime().Format(blockTimeFormat)
+	return u.ToTime().UTC().Format(blockTimeFormat)
```

Comment:
Shouldn't `.UTC()` conversion happen inside `u.ToTime()`?

Reply:
Good call 👍

Reply:
@arnikola what was the reason to move `.UTC()` out of the `UnixNano.ToTime()` again before the merge?

Reply:
It was causing problems with failing tests elsewhere; I'll put out a followup that will since they merit further investigation to double check it's the tests that need updating rather than the code